data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	The pile formed by the colliding teams looks like somewhat like a collective sumo match as the teams attempt to dislodge their opponents from the platform. Within a few seconds, dozens of Kimek have been knocked or outright thrown to the net below, where they are helped out to the sides by workers. Looking through the net, you can see that this is being repeated on each floor. Another whistle blows after enough have fallen out, and the remaining Kimek stop fighting. Heliarch agents return to the ring and organize those left once more into two teams. They depart, and the whistle blows again. This time the fighters have more space to maneuver, so almost a minute passes before any fall.`
+			`	This process repeats several more times until finally, those that remain - less than a tenth of those that entered - are gathered in the center. As they leave the ring, without an announcement of any rankings or score, you become unsure if the game has ended.`
+			`	You feel something tap the top of your head and turn to find that it is an Arach sitting behind you. "Confused, the human visitor is?" their little box translates.`
+			choice
+				`	"Yes. I have no idea what is going on."`
+				`	"I think I somewhat understand, but an explanation would be helpful."`
+			`	They break down the competition for you during what you learn is a brief break. "Numerous, the Kimek are, so many, many athletes competing, they have," they say. "Over a thousand, at first enter each floor. Redone, the teams are, once in half, that number is cut. The true game begins, when one hundred remain."`
+			`	Just then, the remaining contestants reenter the ring almost as if on queue. Now, they are wearing what look like some kind of goggles, perhaps as protective gear. This time, instead of being split into two teams to fight, they are divided into pairs, and each pair moves to one of the small rings painted on the platform.`
+			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`
```

Comment:
Around the outside and down in the middle, on what would otherwise be the floor of your level, underneath the nets that catch the competitors.

The topside of the section that the level below you is fighting on the underside of.

Reply:
Ohh, okay. Thanks for the clarification!

Reply:
If you can word it less awkwardly, I'm all for it. ;)

Thanks btw for catching most of my little slips. I especially tend to drop smaller words when I move things around, but I never notice it unless I sit on it a while before checking again.

Reply:
No problem, thanks for catching my slips too :)